### ISSUE TITLE
Using network time as startup trigger

### ISSUE THREAD

**Issue by manhattan-research-inc** [KenT2/pipresents-next#52]
I'm here at the Barnes Foundation in Philadelphia and am making use of Pi Presents to create a "black box" media player to run a 19-minute video at timed intervals throughout the day with 1 minute of either black screen or message ("Program will start in approx 1 min" or something) in between playback. Goal is to get it running at a set schedule 6 days a week.

The Pi grabs network connection (and thus d/t info) at boot - I'm also at a point where I have it running PP at startup, but it just immediately plays back the video. My issue is that in the event of a power outage or manual interruption, I want the Pi to boot back up and maintain its normal scheduled showtimes, even if it's just sitting and waiting for the next time trigger to happen.

The GUI web editor seems unresponsive to the "edit schedule" command for some reason on my Pi 3 running Stretch and PP ver 1.3.4. I don't know where exactly my time information should be input in any of the JSON docs - are `trigger-start-type` `trigger-start param` and `trigger wait text` my fields of interest as I expected, or am I missing something more obvious?

**Comment by KenT2** [comment]
Hi
This issue applies to pipresents-gapless as you say you are using version 1.3.4
You are correct the editor menu items for the time schedule are not implemented, must improve this.
You need to place a .json file called schedule.json inside the profile directory and edit it with a text editor. There is an example in the /pipresents/pp_resources directory and in the pp_timeofday_1p3 example. 
Read the manual  look at the example and be very careful editing the .json file.

**Comment by manhattan-research-inc** [comment]
Yep, there's a schedule.json file inside the profile with formatting that looks like it follows the example file (someone else was working on this project as well). Do I need to change any parameters in the medialist document to revert towards the schedule JSON as its trigger point, or is it as simple as including the document in the directory? That's what I'm confused about. Alternatively, do you have an ETA for time schedule implementation? Thanks so much!

**Comment by manhattan-research-inc** [comment]
Cool! I'm going to keep working at it in the JSON - let me know if/when you can get a schedule editor up and running through the GUI.
